### ISSUE TITLE
React: Allow attributes to be added to the root node

### ISSUE THREAD

**Issue by argxentakato** [vikejs/vike#3513]
### Description

Since React 17, React's event delegation has caused all synthetic events to be added to the root node. Consequently, some screen readers may incorrectly identify all child elements within the root node as clickable. (See react/react#20895)

This is also called out by the Accessibility Inspector in the Firefox DevTools: "Does not meet WCAG standards for accessibility. Clickable elements must be focusable and should have interactive semantics. [Learn more](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics)"

So, I want to set `role="none"` (or `role="presentation"`) on the root node, but as this is hard-coded, it is not possible.

```tsx
  return escapeInject`<!DOCTYPE html>
    <html${dangerouslySkipEscape(htmlAttributesString)}>
      <head>
        <meta charset="UTF-8" />
        ${headHtmlBegin}
        ${headHtml}
        ${headHtmlEnd}
      </head>
      <body${dangerouslySkipEscape(bodyAttributesString)}>
        ${bodyHtmlBegin}
        <div id="root">${pageHtmlStringOrStream}</div>
        ${bodyHtmlEnd}
      </body>
    </html>`
```
https://github.com/vikejs/vike-react/blob/11df87a1b0eeab5dc244ec3bebb6aac44d8fe532/packages/vike-react/src/integration/onRenderHtml.tsx#L56-L69

Would it be possible to add options that would allow the attributes of the root node to be modified? If not, is there a way to override this?

**Comment by brillout** [comment]
How about a new setting `+rootAttributes`?

**Comment by argxentakato** [comment]
@brillout It's not bad. That's exactly what I want to do.

**Comment by brillout** [comment]
Done: https://vike.dev/rootAttributes (will be live in a moment)
